src/lumigo_tracer/user_utils.py

```diff
@@ -8,7 +8,8 @@
 LUMIGO_REPORT_ERROR_STRING = "[LUMIGO_LOG]"
 MAX_TAGS = 50
 MAX_ELEMENTS_IN_EXTRA = 10
-MAX_TAG_KEY_LEN = MAX_TAG_VALUE_LEN = 50
+MAX_TAG_KEY_LEN = 50
+MAX_TAG_VALUE_LEN = 70
```

Comment:
Please update the docstring of the function `add_execution_tag`.
You'll probably need to update also the docs.